### ISSUE TITLE
Native distribution doesn't work after experiments with accessibility

### ISSUE THREAD

**Issue by igordmn** [JetBrains/compose-multiplatform#1520]
Compose 1.0.0-rc12, OpenJDK 15, Windows 11

Perform in [templates/desktop-template](https://github.com/JetBrains/compose-jb/tree/master/templates/desktop-template):
```
gradlew runDistributable
```

Result:
```
Exception in thread "main" java.awt.AWTError: Assistive Technology not found: com.sun.java.accessibility.AccessBridge
        at java.desktop/java.awt.Toolkit.newAWTError(Unknown Source)
        at java.desktop/java.awt.Toolkit.fallbackToLoadClassForAT(Unknown Source)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
        at java.base/java.util.HashMap$KeySpliterator.forEachRemaining(Unknown Source)
        at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
        at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source)
        at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
        at java.base/java.util.stream.ReferencePipeline.forEach(Unknown Source)
        at java.desktop/java.awt.Toolkit.loadAssistiveTechnologies(Unknown Source)
        at java.desktop/java.awt.Toolkit.getDefaultToolkit(Unknown Source)
        at java.desktop/javax.swing.UIManager.<clinit>(Unknown Source)
        at org.jetbrains.skiko.Setup.init(Setup.kt:25)
        at org.jetbrains.skiko.Setup.init$default(Setup.kt:6)
        at org.jetbrains.skiko.Library.load(Library.kt:92)
        at org.jetbrains.skia.impl.Library$Companion.staticLoad(Library.jvm.kt:12)
        at androidx.compose.ui.ConfigureSwingGlobalsForCompose_desktopKt.configureSwingGlobalsForCompose(ConfigureSwingGlobalsForCompose.desktop.kt:49)
        at androidx.compose.ui.ConfigureSwingGlobalsForCompose_desktopKt.configureSwingGlobalsForCompose$default(ConfigureSwingGlobalsForCompose.desktop.kt:38)
        at androidx.compose.ui.window.Application_desktopKt.application(Application.desktop.kt:76)
        at MainKt.main(main.kt:27)
        at MainKt.main(main.kt)
Caused by: java.lang.ClassNotFoundException: com.sun.java.accessibility.AccessBridge
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Unknown Source)
        ... 22 more
```

**Comment by igordmn** [comment]
It seems it is not a regression. The same crash on Compose `0.4.0`

**Comment by igordmn** [comment]
Reeinstalling OpenJDK didn't help.

Removing
```
C:/Users/user/.accessibility.properties
```
as described [here](https://stackoverflow.com/questions/15260989/exception-in-thread-main-java-awt-awterror-assistive-technology-not-found#comment117751693_15263840), helps, but it is needed to be done on the user machine, not on the machine where we build the application

**Comment by igordmn** [comment]
We need to investigate who creates this file, and why I have a crash now.

I was experimenting with NVDA the day this file was created. I removed it after experiments.
Also I had performed this command:
```
%JRE_HOME%\bin\jabswitch -enable
```

**Comment by lppedd** [comment]
Was this ever solved?  
I just tried to run an `.exe` with Compose `1.5.10-Beta02` and I got the same error.

**Comment by mahozad** [comment]
Is it the same thing as #4126

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6021">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
